sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/AnalysisTest.scala

```diff
@@ -184,7 +184,7 @@ trait AnalysisTest extends PlanTest {
       }
 
       if (e.getErrorClass != expectedErrorClass ||
-        !e.messageParameters.sameElements(expectedMessageParameters) ||
+        e.messageParameters != expectedMessageParameters ||
```

Comment:
nit, Could you fix the indentation by adding two more space here while we are touching this line?